db/seeds.rb

```diff
@@ -1,7 +1,128 @@
-# This file should contain all the record creation needed to seed the database with its default values.
-# The data can then be loaded with the rails db:seed command (or created alongside the database with db:setup).
+# This file should contain all the record creation needed to seed the database
```

Comment:
Were these changes because of rubocop? If so we can probably just disable it from picking up comments as there are tonnes of rails files with long comments that we don't want to have to change every time we edit them.

You could also just delete these comments. You know what goes here and why.